tests/settings/test_generic_file_config.py

```diff
@@ -51,7 +51,3 @@ def test_file_top_path(self):
 
     def test_file_login_user(self):
         self.assertEqual(self.file_config.login_user, "pyiron")
-
-
-if __name__ == "__main__":
-    unittest.main()
```

Comment:
Why is this part removed? I though we need it to run the tests in the continuous integration environment.

Reply:
That's a very good warning!!! It's not necessary on my local machine so I thought I was removing dead code. I'll just burn some CPU minutes and do a brute-force tests by throwing a failing test in one of these suites to see if the CI catches it.

Reply:
Indeed, it's safe to remove the `__main__` boilerplate. I added this to one of the modules that had had the boilerplate stripped and the CI caught it no problem:

```python
    def test_should_fail(self):
        self.assertTrue(False)
```

I'm going to revert the change and resolve the other two instances, but it was a very good call to double check this behaviour!!!